### ISSUE TITLE
[kinesis] transitive dependency on scala 2.12 jar

### ISSUE THREAD

**Issue by pjfanning** [apache/pekko-connectors#479]
I'm not sure if there is anything that can be done to fix this but logging for visibility.

Our Kinesis Connector depends on "software.amazon.kinesis:amazon-kinesis-client" and this depends on "com.kjetland:mbknor-jackson-jsonschema_2.12".

com.kjetland:mbknor-jackson-jsonschema_2.12:1.0.39
software.amazon.glue:schema-registry-serde:1.1.17
software.amazon.kinesis:amazon-kinesis-client:2.5.4

This dependency is brought in transitively and is hardcoded as Scala 2.12 - ie it does not adjust to the Scala version used by Pekko users. We could exclude the dependency and add our own dependency on  "com.kjetland" %% "mbknor-jackson-jsonschema" but there is no Scala 3 release for this jar.

Basically, "com.kjetland" %% "mbknor-jackson-jsonschema" appears not to be actively supported. It hasn't been released since 2020 and has some old Jackson dependencies.

I can raise an issue at https://github.com/mbknor/mbknor-jackson-jsonSchema

**Comment by raboof** [comment]
> We could exclude the dependency and add our own dependency on "com.kjetland" %% "mbknor-jackson-jsonschema" but there is no Scala 3 release for this jar.

Could we use `CrossVersion.for3Use2_13`?

**Comment by pjfanning** [comment]
@jtjeferreira You appear to use the Kinesis connector. Have you run into this issue or do you have any thoughts? Is it possible that we don't use the code in software.amazon.kinesis:amazon-kinesis-client that requires [mbknor-jackson-jsonSchema](https://github.com/mbknor/mbknor-jackson-jsonSchema)?

**Comment by jtjeferreira** [comment]
> @jtjeferreira You appear to use the Kinesis connector. Have you run into this issue or do you have any thoughts? Is it possible that we don't use the code in software.amazon.kinesis:amazon-kinesis-client that requires [mbknor-jackson-jsonSchema](https://github.com/mbknor/mbknor-jackson-jsonSchema)?

I never noticed the use of a `_2.12` dependency because in my projects I exclude the `software.amazon.glue:schema-registry-serde`... 

I found this draft PR https://github.com/awslabs/aws-glue-schema-registry/pull/315 and I think we should not be doing anything and let AWS solve it
